### PR TITLE
react-router - Fix Prompt's `message` typing

### PR DESCRIPTION
**Pull request by kylemh**
## Template
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.) 
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:
If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ReactTraining/react-router/pull/7279

## My own words
- None of the existing tests need to leverage the action param in the callback, so I didnt update them 🤷‍♂️
- I opened [this PR](https://github.com/ReactTraining/react-router/pull/7279) in `react-router` to make the fact reflected in their docs as well.

### CODE REVIEW

**Comment by kylemh** [comment]
I'm not sure why the build failed. Something about storybook-react-router?

```
Error in storybook-react-router
Error: Errors in typescript@3.9 for external dependencies:
../node/globals.d.ts(1075,15): error TS2430: Interface 'Require' incorrectly extends interface 'RequireFunction'.
  Types of property 'cache' are incompatible.
    Type 'Dict<NodeModule>' is not assignable to type '{ [id: string]: NodeModule; }'.
      Index signatures are incompatible.
        Type 'NodeModule | undefined' is not assignable to type 'NodeModule'.
          Type 'undefined' is not assignable to type 'NodeModule'.
```

**Review by eps1lon — CHANGES_REQUESTED** [review]
CI issue looks unrelated. 

Could you go ahead and copy the example from the documentation PR so that we have a test?

**Comment by kylemh** [comment]
@eps1lon I believe I added the test correctly, but I'm not certain.

**Comment by eps1lon** [comment]
@kylemh You need to add the path the to new test to the files in the `tsconfig`: https://github.com/kylemh/DefinitelyTyped/blob/2641a6fd8738f413514e134da0cf10fa8ac1674c/types/react-router/tsconfig.json#L22

**Comment by kylemh** [comment]
Mind if I change that `tsconfig` to use `include`? Feels like we can just include the whole test folder 🤷

File: types/react-router/test/examples-from-react-router-website/README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@ Examples taken from react-router website.
 
 Please keep them in sync and as close as possible to the original ones.
 
-See https://github.com/ReactTraining/react-router/tree/22bd52c901ef0312348c2b02549c7102bd5653ae/packages/react-router-website/modules/examples
+See https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom/examples
```
Comment (eps1lon) [inline types/react-router/test/examples-from-react-router-website/README.md:5]: Could you use a permalink? Otherwise this might get unavailable.

Comment (kylemh) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: How do you mean? Are you suggesting the `master` branch would go away?

Comment (eps1lon) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: No but the path. Using a commit SHA (like before) or a tag would guarantee that the content is available.

Comment (kylemh) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: Right, but wouldn't it also mean the info could be out of date?

I think `master` is better than any `SHA` personally.

Comment (eps1lon) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: Better old than unavailable. With a sha we know whether a type was changed or a mistake was made originally.

Comment (kylemh) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: SHAs can still be force pushed off. I’d argue master remaining the default branch is more permanent even than any SHA in that repositories history.

Nothing is permanent. If nothing is permanent, we might as well be up-to-date.

Comment (eps1lon) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: > SHAs can still be force pushed off. 

Nobody is doing that on master.

Comment (eps1lon) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: If you think that master is better then try replacing the sha in the previous link with `master` and tell me how this improved the link.

Comment (kylemh) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: Well, I changed the link cuz it was broken for me, but I have an idea I think you’ll love.

Comment (kylemh) [reply types/react-router/test/examples-from-react-router-website/README.md:5]: So, I'm not sure what should be done, but leaving it behind seems a little irresponsible 🤔

Comment (kylemh) [reply types/react-router/test/examples-from-react-router-website/README.md]: Okay, I'm hoping this will suffice! Thanks for the reviews.

**Review by eps1lon — APPROVED** [review]

**Pull request merged**